pkg/golinters/exptostd/testdata/exptostd.go

```diff
@@ -0,0 +1,85 @@
+//golangcitest:args -Eexptostd
+package testdata
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/maps"   // want `Import statement 'golang.org/x/exp/maps' can be replaced by 'maps'`
+	"golang.org/x/exp/slices" // want `Import statement 'golang.org/x/exp/slices' can be replaced by 'slices'`
```

Comment:
It can be beneficial of adding a test where `maps` and `slices` already present:

```go
import (
	"fmt"
	"maps"
	"slices"

	"golang.org/x/exp/maps"   // want `Import statement 'golang.org/x/exp/maps' can be replaced by 'maps'`
	"golang.org/x/exp/slices" // want `Import statement 'golang.org/x/exp/slices' can be replaced by 'slices'`
)
```

Reply:
This isn't possible right, you'd have to alias one of them? And the linter currently doesn't try to fix aliased imports so either `"golang.org/x/exp/maps"` is aliased and the linter does nothing, or `"maps"` is aliased and `"golang.org/x/exp/maps"` can safely be changed to `"maps"`.